tests/test_config.py

```diff
@@ -1809,18 +1812,6 @@ def test_quick_search_mode(self):
                                          '--run-config-search-disable',
                                          use_value=False,
                                          use_list=False)
-        self._test_quick_search_with_rcs(args, yaml_content,
```

Comment:
are these invalid now?

Reply:
These are all valid now, so we don't need checks that they are illegal anymore.